python/GafferImageUI/ImageViewUI.py

```diff
@@ -1167,6 +1180,43 @@ def _updateFromPlug( self ) :
 		for i in self.__row[1:]:
 			i.setEnabled( m != "" )
 
+	def __dragEnter( self, tabbedContainer, event ) :
+
+		result = False
+		if isinstance( event.data, Gaffer.Node ) :
+			result = True
+		elif isinstance( event.data, Gaffer.Set ) :
+			if event.data.size() and isinstance( event.data[0], Gaffer.Node ) :
+				result = True
```

Comment:
Rather than repeat this logic in `__drop()`, perhaps we could stick it in a `__dropNode()` function that takes the event and returns the node or `None`?

Reply:
Addressed.